apps/vaporgui/TFOpacityWidget.cpp

```diff
@@ -130,7 +130,7 @@ void TFOpacityMap::paintEvent(QPainter &p)
 void TFOpacityMap::mousePressEvent(QMouseEvent *event)
 {
     emit Activated(this);
-    vec2 mouse = qvec2(event->posF());
+	vec2 mouse = vec2(1.0, 1.0);//qvec2(event->posF());
```

Comment:
Uhh I see here why you were saying the OpacityMap didn't work.

Reply:
#1934

Reply:
My mistake.  Sorry about this.